gobblin-core/src/main/java/gobblin/serde/HiveSerDeWrapper.java

```diff
@@ -48,33 +47,27 @@
   public static final String SERDE_SERIALIZER_TYPE = SERDE_SERIALIZER_PREFIX + "type";
   public static final String SERDE_SERIALIZER_INPUT_FORMAT_TYPE = SERDE_SERIALIZER_PREFIX + "input.format.type";
   public static final String SERDE_SERIALIZER_OUTPUT_FORMAT_TYPE = SERDE_SERIALIZER_PREFIX + "output.format.type";
-  public static final String SERDE_SERIALIZER_FILE_EXTENSION = SERDE_SERIALIZER_PREFIX + "extension";
 
   public static final String SERDE_DESERIALIZER_TYPE = SERDE_DESERIALIZER_PREFIX + "type";
   public static final String SERDE_DESERIALIZER_INPUT_FORMAT_TYPE = SERDE_DESERIALIZER_PREFIX + "input.format.type";
   public static final String SERDE_DESERIALIZER_OUTPUT_FORMAT_TYPE = SERDE_DESERIALIZER_PREFIX + "output.format.type";
-  public static final String SERDE_DESERIALIZER_FILE_EXTENSION = SERDE_DESERIALIZER_PREFIX + "extension";
 
   public enum BuiltInHiveSerDe {
 
-    AVRO(AvroSerDe.class.getName(), AvroContainerInputFormat.class.getName(), AvroContainerOutputFormat.class.getName(),
-        WriterOutputFormat.AVRO.getExtension()),
-    ORC(OrcSerde.class.getName(), OrcInputFormat.class.getName(), OrcOutputFormat.class.getName(),
-        WriterOutputFormat.ORC.getExtension()),
+    AVRO(AvroSerDe.class.getName(), AvroContainerInputFormat.class.getName(),
+        AvroContainerOutputFormat.class.getName()),
+    ORC(OrcSerde.class.getName(), OrcInputFormat.class.getName(), OrcOutputFormat.class.getName()),
     PARQUET(ParquetHiveSerDe.class.getName(), MapredParquetInputFormat.class.getName(),
-        MapredParquetOutputFormat.class.getName(), WriterOutputFormat.PARQUET.getExtension());
+        MapredParquetOutputFormat.class.getName());
```

Comment:
Extensions can be specified using `writer.output.format`. This is used by `FsDataWriter` to construct the file names (if not exists it will default to avro), so one should use this property rather than specifying an extension with the serde.